frontend/views/pages/Home.vue

```diff
@@ -111,12 +111,12 @@ export default {
 }
 
 .logo {
-  width: 8rem;
-  margin: 2rem auto;
+  width: $spacer-xxl;
```

Comment:
`$spacer` should be used for spacing only: margin and padding.
I think using it to define an element size (width, height) is incorrect because it's not the same thing.

I'd leave `8rem`.

Reply:
Next PR..